src/frontend/packages/lib_common/src/cunningham/cunningham-tokens.ts

```diff
@@ -0,0 +1,252 @@
+export const tokens = {
+  themes: {
+    default: {
+      theme: {
```

Comment:
How is this file generated ?
Can we get a flatter structure by removing tokens, themes and default ?
This way we can only use `color={theme.colors['success-500']}` for example
On maybe just export a specific key ?
WDYT ?

Reply:
Yes you'r right.

It is generated here:
https://github.com/openfun/marsha/blob/d2db4aa10b2f5a2d42e66b102aff8948085c14f7/src/frontend/packages/lib_common/package.json#L18

----

We don't have the hand on the generated file but we can add some exports:
https://github.com/openfun/marsha/blob/d2db4aa10b2f5a2d42e66b102aff8948085c14f7/src/frontend/packages/lib_common/src/cunningham/index.ts#L3-L6

----

Easier now:
https://github.com/openfun/marsha/blob/d2db4aa10b2f5a2d42e66b102aff8948085c14f7/src/frontend/packages/lib_components/src/common/Badge/index.tsx#L12